Controller/Complete/Index.php

```diff
@@ -50,23 +50,24 @@ public function execute()
             }
 
             // as AU stack already handle iframe in redirect
-            if ($iframe && $this->_getCurrencyCode() !== CommonUtil::CURRENCY_AUD) {
-                /** @var Page $page */
-                $page = $this->resultFactory->create(ResultFactory::TYPE_PAGE);
-                /** @var Template $block */
-                $layout = $page->getLayout();
-                $block = $layout->createBlock(\Magento\Framework\View\Element\Template::class)
-                    ->setTemplate('Zip_ZipPayment::iframe/iframe_js.phtml');
-                $block->setData('checkoutId', $checkoutId);
-                $block->setData('state', $result);
-                $url = $this->_urlBuilder->getUrl('zippayment/complete')
-                    . '?checkoutId=' . $checkoutId
-                    . '&result=' . $result;
-                $block->setData('redirectUrl', $url);
-                /** @var \Magento\Framework\App\Response $response */
-                $response = $this->getResponse();
-                return $response->setBody($block->toHtml());
-            }
+//            iframe checking is disable until we fix zip checkout js issue to support iframe for all browse
+//            if ($iframe && $this->_getCurrencyCode() !== CommonUtil::CURRENCY_AUD) {
+//                /** @var Page $page */
+//                $page = $this->resultFactory->create(ResultFactory::TYPE_PAGE);
+//                /** @var Template $block */
+//                $layout = $page->getLayout();
+//                $block = $layout->createBlock(\Magento\Framework\View\Element\Template::class)
+//                    ->setTemplate('Zip_ZipPayment::iframe/iframe_js.phtml');
+//                $block->setData('checkoutId', $checkoutId);
+//                $block->setData('state', $result);
+//                $url = $this->_urlBuilder->getUrl('zippayment/complete')
+//                    . '?checkoutId=' . $checkoutId
+//                    . '&result=' . $result;
+//                $block->setData('redirectUrl', $url);
+//                /** @var \Magento\Framework\App\Response $response */
+//                $response = $this->getResponse();
+//                return $response->setBody($block->toHtml());
+//            }
```

Comment:
Git is great for capturing history, so it should be easy to see what used to be here. That might be better than commenting out lines as it might make the code a little harder to read and more messy.

Similar thing[ here.](https://github.com/zipMoney/zip.magento2/pull/33/files#diff-9d8f8bb39dfd46426a24d8fd10c888e246a3a501d2684a7a3af5da46b8c8af58R500) and [here](https://github.com/zipMoney/zip.magento2/pull/33/files#diff-89223417b0d8b4f3be81bd4caf5573ad94c01df7d6e897cb48352e7c6204af25R71). Unless you had something else in mind for keeping them there?